activerecord/test/cases/schema_dumper_test.rb

```diff
@@ -61,7 +61,7 @@ def test_schema_dump
 
   def test_schema_dump_uses_force_cascade_on_create_table
     output = dump_table_schema "authors"
-    assert_match %r{create_table "authors", force: :cascade}, output
+    assert_match %r{create_table "authors", .*?force: :cascade}, output
```

Comment:
We have versioned migrations now!
On Fri, May 20, 2016 at 19:32 Rafael França <notifications@github.com>
wrote:

> In activerecord/test/cases/schema_dumper_test.rb
> <https://github.com/rails/rails/pull/24962#discussion_r64126741>:
>
> > @@ -61,7 +61,7 @@ def test_schema_dump
> >
> >    def test_schema_dump_uses_force_cascade_on_create_table
> >      output = dump_table_schema "authors"
> > -    assert_match %r{create_table "authors", force: :cascade}, output
> > +    assert_match %r{create_table "authors", .*?force: :cascade}, output
>
> Omitting will not have the same problem of changing the default without
> the compatibility layer? If a Rails 4 user load the schema in Rails 5 all
> ids will become bigserial.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/24962/files/c269570a91283e8d623e5715c1f1691e868d7f62#r64126741>
>